.github/workflows/browser-test.yml

```
@@ -0,0 +1,31 @@
+name: Browser Tests
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  browser-tests:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install dependencies
+        run: yarn install
+      - name: Build artifacts
+        run: yarn build
+      - name: Install browser test dependencies
+        run: cd browser-test; npm ci
+      - name: Install Playwright Browsers
+        run: cd browser-test; npx playwright install --with-deps
```

more readable than cd
```suggestion
        run: playwright install --with-deps
        working-directory: browser-test
```